cross-modal-search/Dockerfile

```diff
@@ -1,38 +0,0 @@
-FROM jinaai/jina:2.0.8-perf
```

Comment:
Are we removing the docker flow? The community raised a question regarding this recently:
https://jina-ai.slack.com/archives/C0169V26ATY/p1626675631238300

Reply:
Yes, this does not bring any advantages in my opinion. Why start everything in docker when the executors are dockerized anyways.  Does not make sense in my opinion.